borsh/tests/tests.rs

```diff
@@ -31,9 +31,9 @@ mod compile_derives {
 /// These are full roundtrip `BorshSerialize`/`BorshDeserialize` tests
 #[rustfmt::skip]
 mod roundtrip {
-    mod test_strings; 
+    mod test_strings;
     #[cfg(feature = "ascii")]
-    mod test_ascii_strings; 
+    mod test_ascii_strings;
     mod test_arrays;
     mod test_vecs;
     mod test_tuple;
```

Comment:
add a module with rountrip test below line 40 (test_primitives): 
```rust
#[cfg(feature = "std")]
mod test_ip_addr;
```

it should contain 3 tests: a roundtrip test for `Ipv4Addr`, for `Ipv6Addr` and a 
a roundtrip test for `vec![IpAddr::V4{ ...}, IpAddr::V6{...}, IpAddr::V6{ ...}]`